README.md

```diff
@@ -10,73 +10,19 @@ This project will provide a set of React UI components for the [PatternFly proje
 
 ## Getting Started
 
-First, add `patternfly-react` dependency to your project. If you use Yarn:
+[Getting Started with PatternFly 3](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/README.md)
```

Comment:
would it be worth linking to PF3 / PF4 specific contents in the contributing guide from the README? i.e. I am user - i am only interested in PF4, how do I quickly read the documentation for PF4 instead of PF3. Thanks for updating this!

Reply:
@priley86, I thought about that.  There is a lot of general info that applies to both and I did not want to duplicate the information. But if it is easier to follow, we can have separate guides for PF4 and PF3.

Reply:
@priley86  Take a look at what I have now.  I added some links for "quick" contribution guides for PF3 and PF4 at the top of the contribution md.  I also linked back to the main contribution md from each of those guides also the contributor can get further details.  Let me know if that makes more sense.  If not, we can completely separate them.